libs/feature/map/src/lib/utils/map-utils.service.ts

```diff
@@ -223,7 +223,7 @@ export class MapUtilsService {
     // transform an array of geojson geometries into a bbox
     const totalExtent = record.spatialExtents.reduce(
       (prev, curr) => {
-        const geom = GEOJSON.readGeometry(curr.geometry)
+        const geom = GEOJSON.readGeometry(curr.geometries[0])
```

Comment:
Check if this is correct. Depends on https://github.com/geonetwork/geonetwork-ui/pull/944